library/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ set(src_crypto
     sha1.c
     sha256.c
     sha512.c
+    state.c
```

Comment:
Seems state.c be added into library/.gitignore, which means the state.c will not be exist in the upstream, so where does it come from?

Reply:
It is generated file.